FeedStoreChallenge/CoreDataManager/ManagedFeedImage+CoreDataProperties.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ManagedFeedImage+CoreDataProperties.swift
+//  FeedStoreChallenge
+//
+//  Created by Errol on 27/08/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+extension ManagedFeedImage {
+	@nonobjc public class func fetchRequest() -> NSFetchRequest<ManagedFeedImage> {
+		return NSFetchRequest<ManagedFeedImage>(entityName: "ManagedFeedImage")
+	}
+
+	@NSManaged public var id: UUID
+	@NSManaged public var url: URL
+	@NSManaged public var location: String?
+	@NSManaged public var imageDescription: String?
+	@NSManaged public var cache: ManagedCache
```

Comment:
There's no need to make properties public since `ManagedFeedImage` should be internal
Please remove `public` modifiers
```suggestion
	@NSManaged var id: UUID
	@NSManaged var url: URL
	@NSManaged var location: String?
	@NSManaged var imageDescription: String?
	@NSManaged var cache: ManagedCache
```

Reply:
Made the properties `internal`